artiq/test/coredevice/test_embedding.py

```diff
@@ -1,6 +1,8 @@
 import numpy
 import unittest
 from time import sleep
+from random import randint, seed
+seed(0xdeadbeef)
```

Comment:
Please avoid global state which still has poor reproducibility - create local ``Random`` objects instead. Or just don't use pseudorandom numbers at all.

Reply:
Okay. Of course, if we have something like QuickCheck, it would be better.